core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -555,6 +557,30 @@ public final void setTlsEnabled(final boolean enabled) {
         this.tlsEnabled = enabled;
     }
 
+    /**
+     * Get the host override for TLS SNI Server Name set by {@link #setSniServerName(String)}.
+     *
+     * @return  the sniServerName
+     */
+    public final String getSniServerName() {
+        return sniServerName;
+    }
+
+    /**
+     * Explicitly override the host to use for the TLS SNI server name.
+     *
```

Comment:
Please insert a `<p>` between the paragraphs, otherwise the JavaDoc will contain consecutive sentences in a single paragraph.